manifest.json

```diff
@@ -1,10 +1,10 @@
 {
-	"id": "zoottelkeeper-obsidian-plugin",
-	"name": "Zoottelkeeper Plugin",
-	"version": "0.6.0",
+	"id": "obsidian_autoMOC",
+	"name": "AutoMOC",
+	"version": "0.0.2",
 	"minAppVersion": "0.12.1",
-	"description": "Zoottelkeeper creates and maintains index files in all of your folders in your vault. If you create/delete/move a note, the index files will be updated automatically.",
-	"author": "Akos Balasko",
-	"authorUrl": "https://github.com/akosbalasko",
+	"description": "This plugin automatically creates, maintains and tags MOCs for all your folders. It is based on Zoottelkeeper 0.6.0.",
+	"author": "Akos Balasko, Micha Brugger",

```

Comment:
welcome onboard! :)

Reply:
Haha, I hope you don't take this the wrong way! It was done with the intent of not taking away any credit from you given that it might stay a forked version. Now I feel like I'm taking way too much credit since you did the bulkload of the work.. We can also remove it for now.

Reply:
No no, you can keep yourself in, there are so many improvements in your PR, so I'm glad to share the credits with you!